docs/modules/angular/pages/cookbook/service/portal-dialog-service/index.adoc

```diff
@@ -0,0 +1,214 @@
+= OneCX PortalDialogService
+
+:dynamic_dialog_config_url: https://primeng.org/dynamicdialog#api.dynamicdialog-config
+
+== Overview
+This document aims to showcase how to manage dialogs using OneCX libraries.
+
+== Importing PortalDialogService
+To properly use PortalDialogService, please import it via the `providePortalDialogService`.
+
+.Provide PortalDialogService via the providers array
+[source,javascript]
+----
+import {
+  providePortalDialogService
+} from '@onecx/portal-integration-angular';
```

Comment:
At this point do we still want to publish guides referring to PIA or would now be a good time to move the dialog service and deprecate it in PIA?

Reply:
I agree, however maybe we could wait for libs v6 release candidate for this?

Reply:
There is a story for moving all the useful stuff from PIA such as the dialog service to their correct places. So I would say we can wait until then.